packages/hooks/src/useIsPasskeySupported.ts

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { platformAuthenticatorIsAvailable } from '@simplewebauthn/browser';
+
+const useIsPasskeySupported = () => {
```

Comment:
for the useIsPasskeySupported() - it is simply a wrapper hook around @simplewebauthn/browser which is an external dependencies. I think we shouldn’t test external dependencies